public/ads.index.php

```diff
@@ -1,9 +1,23 @@
 <?php 
 include_once '../bootstrap.php'; 
 
- $stmt = $dbc->query('SELECT * FROM posts');
- $posts=$stmt->fetchAll(PDO::FETCH_ASSOC);
+  $stmt1 = $dbc->query('SELECT * FROM posts');
```

Comment:
This should read something like this.

if ( Input::has('category') ) 
{
    $query = "
        SELECT * FROM `post`
        WHERE category = :" . Input::get('category') ;
}
else if ( Input::has('search'))
{
 do the query for from the search bar
}


This Is how we supposed to be getting thing from form input fields.
Look at the input class that i created to handle form inputs.